twitter_radix/twitter_radix/state/base.py

```diff
@@ -0,0 +1,53 @@
+"""Base state for Twitter example. Schema is inspired by https://drawsql.app/templates/twitter."""
+from typing import Optional
+
+from sqlmodel import Field
+
+import reflex as rx
+
+
+class Follows(rx.Model, table=True):
+    """A table of Follows. This is a many-to-many join table.
+
+    See https://sqlmodel.tiangolo.com/tutorial/many-to-many/ for more information.
+    """
+
+    followed_username: str = Field(primary_key=True)
+    follower_username: str = Field(primary_key=True)
+
+
+class User(rx.Model, table=True):
+    """A table of Users."""
+
+    username: str = Field()
```

Comment:
we don't need the `= Field()` here for default column definition, let's remove them if the old app had it.

Reply:
done